Source/Public/G1TFormatStr.h

```diff
@@ -0,0 +1,1059 @@
+#pragma once
+
+#ifndef G1T_FORMAT_STR_H
+#define G1T_FORMAT_STR_H
+
+// For texture debugging
+// Source: https://github.com/hearhellacopters/G1T
+
+char* getPS3FormatStr(uint8_t format)
```

Comment:
```suggestion
char const* getPS3FormatStr(uint8_t format)
```
For C++23 compatibility, you'll want to make these `const` correct. Otherwise you'll get build errors.